applications/FluidDynamicsApplication/python_scripts/python_solvers_wrapper_fluid.py

```diff
@@ -3,9 +3,9 @@
 import KratosMultiphysics
 
 def CreateSolverByParameters(model, solver_settings, parallelism):
-    
+
```

Comment:
CreateSolverByParameters -> CreateSolverWithParameters

Reply:
This is the same name as in other places, Riccardo came up with it